src/Datadog.Trace/ClrProfiler/Instrumentation.cs

```diff
@@ -66,19 +66,25 @@ public static void Initialize()
                 return;
             }
 
+            Log.Debug("Initialization started.");
+
             try
             {
-                // ensure global instance is created if it's not already
-                _ = Tracer.Instance;
+                Log.Debug("Sending CallTarget integration definitions to native library.");
+                NativeMethods.InitializeProfiler(InstrumentationDefinitions.GetAllDefinitions());
```

Comment:
Should we have this method free the native memory that was allocated for each `NativeCallTargetDefinition`?

Reply:
done in c72171a